## Fibonacci ricorsivo
se tolgo il caso base ad una ricorsione mi viene restituito da python un errore di tipo `RecursionError`

In una ricorsione una funzione chiama s√® stessa e ci sta un caso base